src/cronParser.ts

```diff
@@ -27,12 +23,15 @@ export class CronParser {
   parse(): string[] {
     let parsed: string[];
 
-    var expression = this.expression ?? '';
+    var expression = this.expression ?? "";
 
-    if (expression.startsWith('@')) {
+    if (expression === "@reboot") {
+      // Special handling for @reboot - create a marker array with a special format
+      parsed = ["@reboot", "", "", "", "", "", ""];
+      return parsed;
+    } else if (expression.startsWith("@")) {
```

Comment:
Why is this special handing required - aren't other `@` nicknames supported already like `@daily`?

Reply:
Apologies @bradymholt I should have made more clear. the original PR that implemented @-style nicknames did not fully implement the spec. https://github.com/bradymholt/cRonstrue/pull/318 does not implement `@reboot` which is indeed in the https://www.unix.com/man_page/linux/5/crontab/ specification

Reply:
@reboot needs special treatment because it doesn't 'translate' into a typical 5-part cron sequence